inc/saihash.h

```diff
@@ -54,123 +54,123 @@ typedef enum _sai_native_hash_field_t
      * in case for encapsulated packets.
      * Used for both IPv4 and IPv6
      */
-    SAI_NATIVE_HASH_FIELD_DST_IP,
+    SAI_NATIVE_HASH_FIELD_DST_IP = 0x00000001,
 
     /** Native hash field inner source IP */
-    SAI_NATIVE_HASH_FIELD_INNER_SRC_IP,
+    SAI_NATIVE_HASH_FIELD_INNER_SRC_IP = 0x00000002,
 
     /** Native hash field inner destination IP */
-    SAI_NATIVE_HASH_FIELD_INNER_DST_IP,
+    SAI_NATIVE_HASH_FIELD_INNER_DST_IP = 0x00000003,
 
     /**
      * @brief Native hash field source IPv4.
      *
      * Also, refers to the outer source IPv4
      * in case for encapsulated packets
      */
-    SAI_NATIVE_HASH_FIELD_SRC_IPV4,
+    SAI_NATIVE_HASH_FIELD_SRC_IPV4 = 0x00000019,
```

Comment:
why this enum must bat a this place and not at the end when it was previously there? SRC/DST enums are grouped together even in this configuration so what the point to not make those enums not continuous ?